examples/column-dnd/src/DraggableHeader.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useMemo, useRef } from 'react'
+import { useDrop, useDrag } from 'react-dnd'
+import { getEmptyImage } from 'react-dnd-html5-backend'
+import ItemTypes from './itemTypes'
+import styled from 'styled-components'
+
+const StyledTh = styled.th`
+  cursor: move;
+  opacity: ${props => (props.isDragging ? 0 : 1)};
+`
+
+const DraggableHeader = ({ column, index, reoder }) => {
```

Comment:
reorder

Reply:
Hi @bugzpodder, is something wrong with `reorder`? Maybe the prop name?

Reply:
reoder vs reorder

Reply:
@bugzpodder Incredibly that I didn't see it earlier! 😆 Fixed in commit 224d141dfa4d0a0623348dfb37d90ada1c5eb08e